### PR TITLE
Copied the contents of the directory tutorial3

### PR DESCRIPTION
**Pull request by estelleser**

### CODE REVIEW

File: estelleser/tutorial3/homework3/code/musicstats.sh [hunk 1]
```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
```
Comment (floridop) [inline estelleser/tutorial3/homework3/code/musicstats.sh]: Please remove this file for this submission!

**Comment by floridop** [comment]
Hi Estelle,

Please remove the file musicstats.sh the you created for Homework3 for this submission. I prefer not to have the homework3 in the first submission otherwise everyone can download it from my repo. After that I will approve your pull request.

File: estelleser/tutorial3/homework3/code/musicstats.sh (hunk 1 above)
Comment (estelleser) [reply estelleser/tutorial3/homework3/code/musicstats.sh]: Hi, I removed the homework4 folder as you asked.
While your didn't accepted my pull request, I continued to work on homework3 on this branch. Sorry to disturb you.

Comment (floridop) [reply estelleser/tutorial3/homework3/code/musicstats.sh]: Yes the pull request is kind of a permanent request, if you make changes to a branch that is pending a request they will be reflected in the pull request.
I did not want you to delete the whole homework folder, just your exercise `musicstats.sh`.
However is no big deal, I will accept the pull request anyway so you can proceed forward.

**Pull request merged**